### DISCUSSION TITLE
Ability to pass the --column-statistics option for the BackupCommand

### DISCUSSION THREAD

**Discussion by mattyh88** [spatie/laravel-backup#1100] [Ideas]
Hi, 

I'm currently encountering the following issue when trying to backup databases, when using MySql 8.0.

The dump process failed with exitcode 2 : Misuse of shell builtins : mysqldump: Couldn't execute 'SELECT COLUMN_NAME,                       JSON_EXTRACT(HISTOGRAM, '$."number-of-buckets-specified"')                FROM information_schema.COLUMN_STATISTICS                WHERE SCHEMA_NAME = 'my_database' AND TABLE_NAME = 'my_table_name';': Unknown table 'COLUMN_STATISTICS' in information_schema (1109)

This can be fixed by disabling column statistics, by passing the option --column-statistics to the mysqldump command and settings it's value to zero. However, there's no way to add this option in the current implementation of this package. 

Would be great if that could be added :) 
Thanking you in advance.

**Comment by freekmurze** [comment]
I'm thinking this issue isn't caused by our package. This seems like code from your project `JSON_EXTRACT(HISTOGRAM, '$."number-of-buckets-specified"'`

**Reply by mattyh88** [reply]
Hi Freek, 

Please have a look at this reported issue on serverfault: it has the exact same error message as mine: 
https://serverfault.com/questions/912162/mysqldump-throws-unknown-table-column-statistics-in-information-schema-1109

I don't have the issue on my local machine, where I'm running Mysql 5.7
It only occurs in our production environment, where we are running Mysql 8
